src/app/Caches/Create/CreateCacheWizard.tsx

```diff
@@ -151,6 +157,78 @@ const CreateCacheWizard = (props) => {
         history.push('/');
     };
 
+    const onGoToStep = (id, prevId) => {
+        console.log(id, prevId);
+    };
+
+    const getNextStep = (activeStep, callback) => {
+        if (activeStep.name === t('caches.create.getting-started.nav-title')) {
```

Comment:
I won't use the name with the translation here. We need an identifier for each step and the label is displayed somewhere else.